jose-jwt/jwa/EcdhKeyManagementWinWithAesKeyWrap.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+
+namespace Jose
+{
+    public class EcdhKeyManagementWinWithAesKeyWrap : EcdhKeyManagementUnix
```

Comment:
This should fix a couple of your tests.

```suggestion
    public class EcdhKeyManagementWinWithAesKeyWrap : EcdhKeyManagementWin
```